sqlite.go

```diff
@@ -446,6 +491,10 @@ func (res *stmtResult) LastInsertId() (int64, error) { return res.lastInsertID,
 func (res *stmtResult) RowsAffected() (int64, error) { return res.rowsAffected, nil }
 
 func (s *stmt) QueryContext(ctx context.Context, args []driver.NamedValue) (driver.Rows, error) {
```

Comment:
you also want all these `*stmt` methods to check whether the stmt itself was closed (via stmt.Close) before touching the sqliteh.Stmt

Reply:
How does that work with statements with `s.persist == true`?

Reply:
After a bunch of testing, I think the right thing to do is to only check if the `*stmt` is closed if `s.persist==false`; just amended to do that. PTAL?